src/test/resources/com/puppycrawl/tools/checkstyle/checks/InputAvoidEscapedUnicodeCharacters.java

```diff
@@ -102,4 +102,17 @@ void foo() {
 	private String notAUnicodeEscaped2 = "\\\\u1234";
 
 	private String onlyEscaped = "\\\u1234";
+
+	private String controlC0 = ""
+			+ "\u0000\u0001\u0002\u0003\u0004\u0005\u0006\u0007\u0008\u0009\n\u000b\u000c\r\u000e\u000f"
```

Comment:
Is there a reason for `\n` and `\r` and not their unicode equivalent like the others?